webapp/index.html

```diff
@@ -8,13 +8,14 @@
 	<title>UI5Lab Sample Browser</title>
 
 	<script id="sap-ui-bootstrap"
-			src="../../../resources/sap-ui-core.js"
-			data-sap-ui-libs="sap.m,sap.f"
+			src="/resources/sap-ui-core.js"
+			data-sap-ui-libs="sap.m, sap.f, sap.ui.layout"
```

Comment:
minor: you only need to specify the libs here that are used in the html page, the other libs are loaded from the manifest:
sap.m should be enough if needed at all